source/quickstart.txt

```diff
@@ -1,12 +1,235 @@
+
+         cluster0:
+         mongodb://clusterAdmin:superSecret@clusterOne01.fancyCorp.com:20020,clusterOne02.fancyCorp.com:20020,clusterOne03.fancyCorp.com:20020
+         cluster1:
+         mongodb://clusterAdmin:superSecret@clusterTwo01.fancyCorp.com:20020,clusterTwo02.fancyCorp.com:20020,clusterTwo03.fancyCorp.com:20020
+
+      For more details, see :ref:`c2c-connecting`.
+
+Initialize ``monogsync``
+------------------------
+
+``monogsync`` must create an initial connection to the source and
+destination clusters before it can start to sync data. To create the
+initial connection, issue the following command on a single line (the
+command is reformated here for clarity):
+
+.. code-block:: shell
+
+   ./bin/mongosync --cluster0 mongodb://clusterAdmin:superSecret@clusterOne01.fancyCorp.com:20020,
+                              clusterOne02.fancyCorp.com:20020,
+                              clusterOne03.fancyCorp.com:20020
+                   --cluster1 mongodb://clusterAdmin:superSecret@clusterTwo01.fancyCorp.com:20020,
+                              clusterTwo02.fancyCorp.com:20020,
+                              clusterTwo03.fancyCorp.com:20020
+
+Initialization Notes
+~~~~~~~~~~~~~~~~~~~~
+
+- When ``mongosync`` first connects to the source and destination
+  clusters it is in the :ref:`IDLE <c2c-states-descriptions>` state.
+- ``mongosync`` does not synchronize data until it receives the
+  :ref:`start <c2c-api-start>` command. 
+- Designate the source and destination clusters with the :ref:`start
+  <c2c-api-start>` command. "cluster0" and "cluster1" are just labels,
+  either cluster can be ``cluster0`` or ``cluster1``.
+
+Synchronize Data Between Clusters
+---------------------------------
+
+The command interface for ``mongosync`` is an HTTP server that
+publishes a REST API. To control ``mongosync``, use the API endpoints.
+
+The :ref:`start <c2c-api-start>` command initiates data synchronization.
+To start synchronizing, use ``curl`` or a similar program to issue the
+:ref:`start <c2c-api-start>` command:
+
+.. literalinclude:: /includes/api/requests/start.sh
+   :language: shell
+
+Once started, ``mongosync`` will synchronize existing data on the
+source cluster with the destination cluster. After the initial sync has
+completed, new writes to the source cluster will be synchronized with
+the destination cluster.
+
+The API documentation provides details on using the following endpoints:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Endpoint
+     - Description
+ 
+   * - :ref:`c2c-api-commit`
+     - Commits the synchronization operation to the destination
+       cluster.
+   * - :ref:`c2c-api-pause`
+     - Pauses the current synchronization operation.
+   * - :ref:`c2c-api-progress`
+     - Returns the status of the synchronization process.
+   * - :ref:`c2c-api-resume`
+     - Resumes a paused synchronization session based on data stored on
+       the destination cluster.
+   * - :ref:`c2c-api-reverse`
```

Comment:
Just wanted to mention that the rendering on staging for this option looked wrong:
![Screen Shot 2022-06-06 at 10 52 45 AM](https://user-images.githubusercontent.com/3639158/172186093-85afda6d-0880-4f63-b473-075122569753.png)

Reply:
Thanks - I think the staging link is out of date. I'll update it

Reply:
Understand that this is alphabetized, but /commit as the first API threw me off. can we match the ordering under the API dropdown? 

under the reference -> API page we followed the order:
/start
/progress
/pause
/resume
/commit
/reverse

Reply:
Done